src/services/ThankYouEmailProcessor.ts

```diff
@@ -73,6 +76,18 @@ export class ThankYouEmailProcessor {
 			  govuk_signin_journey_id: f2fSession.clientSessionId,
 		  });
 
+			//Initialise Yoti Service base on SessionClietID
+			const clientConfig = getClientConfig(this.environmentVariables.clientConfig(), f2fSession.clientId, this.logger);
+
+			if (!clientConfig) {
+				this.logger.error("Unrecognised client in request", {
+					messageCode: MessageCodes.UNRECOGNISED_CLIENT,
+				});
+				return new Response(HttpCodesEnum.BAD_REQUEST, "Bad Request");
+			}
+
+			this.yotiService = YotiService.getInstance(this.logger, this.environmentVariables.yotiSdk(), this.environmentVariables.resourcesTtlInSeconds(), this.environmentVariables.clientSessionTokenTtlInDays(), this.YOTI_PRIVATE_KEY, clientConfig.YotiBaseUrl);
```

Comment:
[nit] can we please break up this line? Apparently it's 262 chars and the limit is 140

Reply:
Done